1.cellprofiler_ic/README.md

```diff
@@ -0,0 +1,15 @@
+## Perform illumination correction and save corrected images
```

Comment:
Consider making this a header-1 as it's the first header within the document. This is sometimes important for web accessibility (for ex. with screen-readers).
```suggestion
# Perform illumination correction and save corrected images
```

Reply:
Oops that was a typo. Fixed!